packages/did/src/IdenityRegistry.js

```diff
@@ -1,7 +1,7 @@
 const tic = require('tic.api.js');
 const mamClient = require('mam.tools.js');
 
-const { encodeToDid, decodeFromDid } = require('./mnid');
+const { encodeTangleId, decodeTangleId } = require('./mnid');
```

Comment:
Why would you change?

Reply:
Tangle is one of the blockchain implementations, fitting the requirement of DID.

Reply:
How about rename them to the `encodeDid` and `decodeDid`? 🤔

Reply:
Either introduce new layer for encapsulating Tangle or rename back to encodeToDid/decodefromDid. I prefer the later.

Reply:
Okay, I'll change them as following example:
     - encode() -> encodeToMnid()
     - decode() -> decodeFromMnid()
     - encodeToDid() -> encodeToDid()
     - decodeFromDid() -> decodeFrom()

Reply:
The function names are updated. 👍